src/Entity/Board.php

```diff
@@ -38,16 +45,22 @@ class Board
      */
     private $advisors;
```

Comment:
Can you name this attribute as a singular noun, since you'll only get one advisor when searching

Reply:
it's a many to many relation, you'll get an array of advisors. I think the plural is relevant